### ISSUE TITLE
align download and execute_batch

### ISSUE THREAD

**Issue by jdries** [Open-EO/openeo-python-client#240]
I find myself switching between download and execute_batch quite often, but need to adjust the output format parameters.
Would it be possible to align this, without breaking existing code?

**Comment by soxofaan** [comment]
Indeed, I also these small inconsistencies in sync vs batch execution annoying

**Comment by soxofaan** [comment]
this could be part of the solution: https://github.com/Open-EO/openeo-python-client/issues/238
